mods/d2k/chrome/ingame-observer.yaml

```diff
@@ -494,7 +494,7 @@ Container@OBSERVER_WIDGETS:
 						Container@COMBAT_STATS_HEADERS:
 							X: 0
 							Y: 0
-							Width: 735
+							Width: 820
```

Comment:
I fear this will cause #17059 again.

Reply:
We might need to squeeze out some more space from the other items or remove some of them.

Reply:
Good spotting, I can confirm the regression.

Reply:
One option could be to abbreviate "Units Killed/Lost" and "Bldg Killed/Lost" further to "U. Killed/Lost" and "B. Killed/Lost" with a tooltip showing the full Units / Buildings.

Reply:
We could also combine both into a single `Killed/Lost` and then show the individual `Units/Buildings Killed/Lost` in the tooltip.

Reply:
I also got rid of the abbreviation while I was at it:

![image](https://user-images.githubusercontent.com/756669/102385943-bc4e6800-3fce-11eb-9840-25821f5c012e.png)